### PR TITLE
Adds several KA mods and allows mining borgs to install any KA mods

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Adds a KA mod that does damage in an AoE, found only in necropolis chests.
rscadd: You can now install any KA mods into a mining cyborg, though its mod capacity is slightly lower.
tweak: Mining cyborgs now have a crowbar to remove mods from their KA.
tweak: Kinetic Accelerators now have a larger mod capacity, though most mods can still only be installed the same amount of times as they can currently.
tweak: You can now hit modkits with Kinetic Accelerators to install that modkit into the accelerator. Mining cyborgs are unable to do so.
rscadd: Added white and adjustably-coloured tracer round mods to the mining equipment vendor, so you can see where your projectile is going(and look cool).
rscadd: Added the super and hyper chassis mods to the mining equipment vendor, so you can use excess mod space to have a cool-looking KA.
/:cl:

Currently, the AoE mod does 25% of the blast's damage to mobs that weren't hit directly(max of 75% if you manage to find three and use a good 90% capacity on them)

### CODE REVIEW

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -194,23 +194,29 @@
 
 	return 1
 
-/obj/item/borg/upgrade/hyperka
-	name = "mining cyborg hyper-kinetic accelerator"
-	desc = "A satchel of holding replacement for mining cyborg's ore satchel module."
```
Comment (GunHog) [inline code/game/objects/items/robot/robot_upgrades.dm]: ;_; How did I not catch that????????

**Comment by GunHog** [comment]
Since you are touching borg upgrades, could you make it such that KA mods produced from the protolathe work equally on a borg's KA as they would a human one?

**Comment by ChangelingRain** [comment]
I would but that seems better for another PR, honestly.

**Comment by GunHog** [comment]
Okay :(  At least you considered it

**Comment by Ergovisavi** [comment]
If I get hit by this shit while I'm using the crusher I swear to god

**Comment by ChangelingRain** [comment]
rip ergo killed by friendly fire

**Comment by GunHog** [comment]
Yeeeeeeeeey!

**Pull request merged**